vignettes/pagebreak.Rmd

```diff
@@ -0,0 +1,153 @@
+---
+title: "Add a pagebreak in Rmarkdown document"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{pagebreak}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+# Usage
+
+Adding a pagebreak in document was always possible using custom output specific syntax in a rmarkdown file but one drawback was the compatibility with several output format.
+
+Since rmarkdown >= 1.15 and with RStudio >= 1.2 (or with pandoc >= 2.0), it is possible to add a `\newpage` or `\pagebreak` command in a new line to include a pagebreak in any of these formats: `pdf_document()`, `html_document()`, `word_document()` and `odt_document()`. 
+
+```md
+# Header 1
+
+Some text
+
+\newpage
+
+# Header 2 on a new page
+
+Some other text
+
+\pagebreak
+
+# Header 3 on a third page
+
+```
+
+rmarkdown will convert those commands in the correct output format syntax using a [lua filter](#lua-filter) during pandoc conversion. 
+
+# Using with PDF/ latex documents {#pdf}
+
+As the commands are the ones already used in latex syntax, this works as expected in a tex output document, and thus with pdf. Adding a pagebreak was already possible with rmarkdown when output is `pdf_document()` or `latex_document()`, without any restriction about the version of pandoc.
+
+# Using with HTML documents {#html}
+
+A `\newpage` or `\pagebreak` command in a rmarkdown document with output as HTML will be converted by default in this html code with inline style using CSS rule [`page-break-after`](https://www.w3schools.com/cssref/pr_print_pageba.asp)
```

Comment:
IMO, it would be better to link to an official documentation like MDN. W3Schools has many outdated pages (on this subject, see [this SO Meta discussion](https://meta.stackoverflow.com/questions/280478/why-not-w3schools-com)).
The MDN page is https://developer.mozilla.org/en-US/docs/Web/CSS/page-break-after and explain that we should use `break-after: page` now (obviously, `page-break-after` still works and I confess that I use it).

Reply:
Thanks ! I'll change the link. 

Do you think we should change the lua filter to use `break-after: page` in inline style ?

Reply:
I think `break-after: page;` should be promoted while explaining that it replaces the older CSS property `page-break-after`.

Reply:
Is this already well supported ? I am always unsure with this.
On https://caniuse.com/#search=page-break, it seems the new `break-*` alias is not well supported yet... 🤔

Reply:
Okay, maybe the best solution is to mention both and pointing to caniuse (it is always up to date)